test/e2e/e2e_suite_test.go

```diff
@@ -5,50 +5,41 @@ package e2e
 
 import (
 	"context"
-	"flag"
-	"fmt"
-	"os"
-	"os/user"
-	"path/filepath"
-	"strconv"
 	"testing"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/open-cluster-management/governance-policy-framework/test/common"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/klog"
 )
 
 var (
-	userNamespace          string
-	clusterNamespace       string
-	clientHub              kubernetes.Interface
-	clientHubDynamic       dynamic.Interface
-	clientManaged          kubernetes.Interface
-	clientManagedDynamic   dynamic.Interface
-	gvrPod                 schema.GroupVersionResource
-	gvrPolicy              schema.GroupVersionResource
-	gvrIamPolicy           schema.GroupVersionResource
-	gvrCertPolicy          schema.GroupVersionResource
-	gvrConfigurationPolicy schema.GroupVersionResource
-	gvrPlacementBinding    schema.GroupVersionResource
-	gvrPlacementRule       schema.GroupVersionResource
-	gvrRole                schema.GroupVersionResource
-	gvrCRD                 schema.GroupVersionResource
-	gvrNS                  schema.GroupVersionResource
-	gvrK8sRequiredLabels   schema.GroupVersionResource
-	kubeconfigHub          string
-	kubeconfigManaged      string
-	defaultTimeoutSeconds  int
+	clusterNamespace = "managed"
 
-	defaultImageRegistry string
+	defaultTimeoutSeconds int
```

Comment:
Why is this variable defined here rather than use `common.defaultTimeoutSeconds` directly? I have the same question for `userNamespace`.

Reply:
Good questions. First, these things appear a ton in the tests themselves. So although I could update all of those occurrences to prepend `common.`, I'd rather not - in addition to making this PR huge, it would make some lines incredibly long.

By being lazy and requiring `userNamespace = common.UserNamespace` in the BeforeSuite, it also provides an obvious point where a developer could interject their own custom value as needed. For example, the tests here specify "managed" instead of `common.ClusterNamespace`.

Devs could also use command line flags to set things like userNamespace, and it's helpful to have those flags defined in the common package. BUT, as a quirk of flag parsing and ginkgo, those flags can't be parsed in `init` functions, so they won't have the right values until the BeforeSuite function. That's why we can't for example do `var userNamespace = common.UserNamespace`.

Reply:
Thanks for the explanation!